### DISCUSSION TITLE
Cant sync

### DISCUSSION THREAD

**Discussion by hibenji** [Chia-Network/chia-blockchain#6944] [Support]
I have serious issues with syncing, i have wait about 3 days to sync 3k blocks.
I used the GUI until now and now i tried to use CLI (both on windows).
On the CLI i gets these issues as seen in the picture, i have no idea why or how and i think it is keeping me from syncing.
![image](https://user-images.githubusercontent.com/65447501/123134568-344f8c00-d451-11eb-9433-c0a6484b6975.png)

Thanks for any help

**Comment by hibenji** [comment]
This problem has been solved, my pc time was out of sync.

**Comment by BrandtH22** [comment]
@hibenji , based on your previous message it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
